src/components/onboarding/sign-in/index.js

```diff
@@ -84,42 +69,30 @@ class SignIn extends React.Component {
   renderForgot = () => {
     return (
       <div css={style.forgot}>
-        <LinkButton css={{ justifyContent: 'flex-end' }} to={ROUTE.PATH.HOME}>
-          <div css={style.forgotText}>Forgot?</div>
+        <LinkButton to={ROUTE.PATH.HOME}>
+          <div css={style.forgot__text}>{'Forgot?'}</div>
```

Comment:
Sorry, let's do `_` for element, `__` for modifier

Reply:
Also no need to use it twice: `help_link`, `help_linkText`. Not `help_link_text`.

Reply:
You would need to explain it better to us. For me, it is just a mess right now.